src/cascadia/TerminalControl/IControlSettings.idl

```diff
@@ -61,5 +61,6 @@ namespace Microsoft.Terminal.Control
         Boolean SoftwareRendering { get; };
         Boolean ShowMarks { get; };
         Boolean UseBackgroundImageForWindow { get; };
+        Boolean ReloadEnvironmentVariables { get; };

```

Comment:
honestly I think we can make the diff even smaller. I don't _think_ the control actually needs to know if `ReloadEnvironmentVariables` is set at all. You should be able to revert the changes to `IControlSettings`, `TerminalSettings.*`, and just check `_settings.GlobalSettings().ReloadEnvironmentVariables()` in `TerminalPage`

Reply:
You're right. I saw a `settings` available to me in `TerminalPage` and thought "I'll use that" :)